### DISCUSSION TITLE
Individual styling per column

### DISCUSSION THREAD

**Discussion by matthewrichter** [Power-Components/livewire-powergrid#953] [Q&A] [answered]
Hi

I know I can add styling in the theme folder under Tailwind. But it effects everything the same.

for example if I add text-centre to tbody, it centers the text for everything. Am I able to text-left one column and then text-center another?

regards
Matt

**Comment by abhaypithadiya** [comment, marked as answer]
Hey!

You can use headerAttribute for class or style to the column header and bodyAttribute for class or style to each table row in this column.

```
Column::make('USER', 'user_name')
    ->headerAttribute('text-center', 'color:red')
    ->bodyAttribute('text-center', 'color:red')
```

Hope this helps!
